charts/timescaledb-single/Chart.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.30.0
+version: 0.30.1
```

Comment:
LGTM!  Thanks for the PR.  

The only nit is updating this to `0.31.0` when you can.